driver/camera.c

```diff
@@ -1094,16 +1106,30 @@ esp_err_t camera_init(const camera_config_t* config)
         }
         s_state->fb_bytes_per_pixel = 1;       // frame buffer stores Y8
     } else if (pix_format == PIXFORMAT_YUV422 || pix_format == PIXFORMAT_RGB565) {
-        s_state->fb_size = s_state->width * s_state->height * 2;
```

Comment:
looking at the code, please revert this whole change. the camera will work fine without it. You should do the conversion to 888 in your code. Since a lot of the work happens in DMS Interrupts, you writing 50% more bytes can cause CPU starvation. In other words, I've seen it fully utilising the CPU inside interrupts.

Reply:
Hi, thanks for all suggestions, in the next days I will implements.